frontend/src/old-pages/Configure/Cluster.tsx

```diff
@@ -258,7 +258,7 @@ function VpcSelect() {
           <Select
             disabled={editing}
             // @ts-expect-error TS(2322) FIXME: Type '{ label: JSX.Element; value: any; }' is not ... Remove this comment to see the full error message
-            selectedOption={vpcToDisplayOption(findFirst(vpcs, (x: any) => {return x.VpcId === vpc}))}
+            selectedOption={vpcToDisplayOption(findFirst(vpcs, x => x.VpcId === vpc))}
```

Comment:
suggestion: can we apply the same fix you did `JobSumitDialog.tsx` to the `itemToOption` function?

Reply:
Will fix in separate PR